llvm-bazel/.bazelrc

```diff
@@ -62,6 +62,48 @@ build:generic_gcc --copt=-Wno-misleading-indentation --host_copt=-Wno-misleading
 # Use `-Werror` for GCC to make sure warnings don't slip past.
 build:generic_gcc --copt=-Werror --host_copt=-Werror
 
+###############################################################################
+# Windows specific flags for building with VC.
+###############################################################################
+
+build:msvc --copt=/WX     # Treat warnings as errors...
+
+# ...but disable the ones that are violated
+build:msvc --copt=/wd4624 # destructor was implicitly defined as deleted
+build:msvc --copt=/wd4244 # possible loss of data
+build:msvc --copt=/wd4005 # macro redefinition
+build:msvc --copt=/wd4141 # inline used more than once
+build:msvc --copt=/wd4267 # initializing: possible loss of data
+build:msvc --copt=/wd4273 # multiple definitions with different dllimport
+build:msvc --copt=/wd4319 # zero-extending after complement
+build:msvc --copt=/wd4804 # comparisons between bool and int
+build:msvc --copt=/wd4805 # comparisons between bool and int
+
+build:msvc --linkopt=/WX # Treat warnings as errors...
+
+# ...but disable the ones that are violated.
+build:msvc --linkopt=/IGNORE:4001 # no object files
+build:msvc --linkopt=/IGNORE:4217 # mismatch import/export declspec
+
+# Enables unix-style runfiles link trees (requires symlink permission).
+# See: https://blogs.msvc.com/msvcdeveloper/2016/12/02/symlinks-msvc-10/
+# Generally: Enable Developer Mode in the Developer Settings page of the
+# system settings.
+build:msvc --experimental_enable_runfiles
+build:msvc --copt=/arch:AVX
```

Comment:
We don't pass /arch:avx upstream, and I guess this will narrow the targets your builds will run on. As long as that's intentional, I suppose it's okay.